fdbclient/NativeAPI.actor.cpp

```diff
@@ -825,13 +826,15 @@ ACTOR Future<Void> monitorCacheList(DatabaseContext* self) {
 ACTOR static Future<Void> handleTssMismatches(DatabaseContext* cx) {
 	state Reference<ReadYourWritesTransaction> tr;
 	state KeyBackedMap<UID, UID> tssMapDB = KeyBackedMap<UID, UID>(tssMappingKeys.begin);
+	state KeyBackedMap<Tuple, std::string> tssMismatchDB = KeyBackedMap<Tuple, std::string>(tssMismatchKeys.begin);
```

Comment:
Just wondering, does it work to do `= {tssMismatchKeys.begin}` here for initialization?

Reply:
It compiled?

Reply:
List initialization works here, see [this stackoverflow question](https://stackoverflow.com/questions/18222926/why-is-list-initialization-using-curly-braces-better-than-the-alternatives)

But I believe the following would also work:
```suggestion
	state KeyBackedMap<Tuple, std::string> tssMismatchDB(tssMismatchKeys.begin);
```
```suggestion
	state KeyBackedMap<Tuple, std::string> tssMismatchDB = KeyBackedMap<Tuple, std::string>(tssMismatchKeys.begin);
```